launcher/src/main/java/org/apache/spark/launcher/SparkLauncher.java

```diff
@@ -64,6 +64,10 @@
   /** Configuration key for the number of executor CPU cores. */
   public static final String EXECUTOR_CORES = "spark.executor.cores";
 
+  public static final String PYSPARK_DRIVER_PYTHON = "spark.pyspark.driver.python";
```

Comment:
I know, but this is a public as in "it's actually exposed to users" API. I'd rather avoid adding more and more config keys to this class - especially when, like these, they're probably not gonna be commonly used.

Inside Spark, you can add the constants I suggested previously. But I'd prefer to keep these new fields in `SparkLauncher` not public.